docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -196,11 +336,242 @@ To process the received command on the server side:
+
+   ```
+   mc_client_destroy(g_client_h);
+   ```
+
+To process the received command on the server side:
+
+1. Create the media controller server handle using the `mc_server_create()` function:
+
+   ```
+   ret = mc_server_create(&g_server_h);
+   ```
+
+2. Define the callback that is invoked when the server receives the command.
+
+   For example, to define a callback for playback state change commands:
+
+   ```
+   void
+   playback_action_cmd_received_cb(const char* client_name, const char *request_id, mc_playback_action_e action, void *user_data)
+   {
+       dlog_print(DLOG_DEBUG, LOG_TAG, "Client Name: %s, Request Id: %s, Playback action: %d\n", client_name, request_id, action);
+   }
+   ```
+
+3. Register the callback:
+
+   - To register a callback for playback state change commands, use the `mc_server_set_playback_action_cmd_received_cb()` function.
+   - To register a callback for playback position change commands, use the `mc_server_set_playback_position_cmd_received_cb()` function.
+   - To register a callback for shuffle mode change commands, use the `mc_server_set_shuffle_mode_cmd_received_cb()` function.
+   - To register a callback for repeat mode change commands, use the `mc_server_set_repeat_mode_cmd_received_cb()` function.
+   - To register a callback for played item, playback state and playback position change commands in playlist, use the `mc_server_set_playlist_cmd_received_cb()` function.
+   - To register a callback for a custom command, use the `mc_server_set_custom_cmd_received_cb()` function.
+
+   For example, to register a callback for playback state change commands:
+
+   ```
+   ret = mc_server_set_playback_action_cmd_received_cb(g_mc_server, playback_action_cmd_received_cb, NULL);
+   ```
+
+4. Destroy the media controller server handle using the `mc_server_destroy()` function, when media controller server handle is no longer needed.
+
+   ```
+   mc_server_destroy(g_server_h);
+   ```
+
+<a name="send_cmd_reply"></a>
+To send the reply of completed command on the server side:
+
+1. Send the reply of completed command using the `mc_server_send_cmd_reply()` function with the request id of the command in the third parameter and the result of the command in fourth parameter:
+
+   ```
+   int result_code = 0;
+
+   ret = mc_server_send_cmd_reply(g_server_h, client_name, request_id, result_code, NULL);
+   ```
+
+To receive the reply of completed command on the client side:
+
+1. Define the callback that is invoked when the client receives the reply.
+
+   ```
+   void
+   cmd_reply_received_cb(const char *server_name, const char *request_id, int result_code, bundle *data, void *user_data)
+   {
+       dlog_print(DLOG_DEBUG, LOG_TAG, "Server Name: %s, Request Id: %s, Result Code: %d\n", server_name, request_id, result_code);
+   }
+   ```
+
+2. Register the callback:
+
+   ```
+   ret = mc_client_set_cmd_reply_received_cb(g_client_h, cmd_reply_received_cb, NULL);
+   ```
+
+**Note**
```

Comment:
From line 456 to line 458, please add "> " in front of the lines.
```
> **Note**
>
> This feature supports Tizen 4.0 and higher for Mobile.
```

Reply:
I added "> " in front of the Note.